modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,9 +21,9 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
 endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
-= Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
```

Comment:
No markup or backticks in headings. See the [OCP repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)